File: birthday/submissions/accepted/birthdayConstant.cpp
#include <iostream>

using namespace std;

/**
 * TOOD: 
 * 
 * A: a non-negative integer
 */
int solve(int A) {
    // YOUR CODE HERE

    return (A * (A-1) * (2*A -1) / 6) +3;
}

int main() {
    int T;
    cin >> T;
    for (int i = 0; i < T; i++) {
        int A;
        cin >> A;
        cout << solve(A) << '\n';
    }
}

File: birthday/submissions/accepted/birthdayConstant.py
def solve(A: int) -> int:
    """
    TODO: 
    
    A: a non-negative integer
    """
    # YOUR CODE HERE
    return A*(A-1)*(2*A-1) // 6 +3


def main():
    T = int(input())
    for _ in range(T):
        temp = input().split()
        A = int(temp[0])
        print(solve(A))

if __name__ == '__main__':
    main()
